src/main/resources/altinn-rettigheter-proxy-client.properties

```diff
@@ -0,0 +1 @@
+klient.versjon=${project.version}
```

Comment:
Hvor kommer project.version fra? Er den bare alltid tilgjengelig via pom.xml?

Reply:
Det maven som gjør jobben her. Når du legger til en `${name.of.a.maven.variable}` i en fil i resource mappen, skal maven oppdatere filen med tilsvarende verdien (variabel og verdi er avhengig av hvilken maven plugin du bruker, men vi bruker et ganske klassisk opplegg med compile/test/build jar). I tillegg til det har vi lagt til `<resource>` i _build_ fasen i pom filen. Dette gjør at resource filene blir kopiert i Jar filen.